fintrack/accounts/management/commands/populate_data.py

```diff
@@ -0,0 +1,35 @@
+import datetime
+
+
+from django.core.management.base import BaseCommand
+
+
+from accounts.models import  Account
+from banks.models import Bank, Branch
+# from transactions.transaction_helper import create_transaction, delete_transaction
+from users.models import User
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **kwargs):     
+        # user, _ = User.objects.get_or_create(username='hamza65', password='hamza')
```

Comment:
why these are commented out?

Reply:
I was using this just for testing purpose